hummingbot/market/binance/binance_market.pyx

```diff
@@ -483,6 +483,7 @@ cdef class BinanceMarket(MarketBase):
                 client_order_id = tracked_order.client_order_id
                 if isinstance(order_update, Exception):
                     if order_update.code == 2013 or order_update.message == "Order does not exist.":
+                        order_type = OrderType.LIMIT if order_update["type"]=="LIMIT" else OrderType.MARKET
```

Comment:
fix spacing `] == "LIMIT"`

Reply:
I just reused the code which was there before. Fixed this and that one too.